package.json

```
@@ -29,6 +28,7 @@
   },
   "homepage": "https://github.com/Mogztter/asciidoctor-pdf.js#readme",
   "dependencies": {
+    "@asciidoctor/cli": "file:../asciidoctor-cli.js",
```

```suggestion
    "@asciidoctor/cli": "github:Mogztter/asciidoctor-cli.js#issue-64-cli-api",
```

Then locally in the `asciidoctor-cli.js` folder of your machine, run `npm link`. And in this project directory, run `npm link @asciidoctor/cli`.

---

This is pure awesomeness, I didn't know that, thanks :+1: